composer.json

```diff
@@ -6,12 +6,13 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "2.16.4",
         "friendsofredaxo/linter": "1.2.6",
-        "phpstan/phpstan": "0.12.31",
+        "nikic/php-parser": "4.6.0",
```

Comment:
was ist das konkrete problem, dass wir hier die version fixieren müssen?

wird der dependabot das wieder versuchen up-zu-daten?

Reply:
Ich hab es nur am Rand mitbekommen im Symfony Slack. Wenn ich es richtig verstehe, interpretiert die neuere Version bereits zum Teil den Code als PHP8-Code, was wohl insbesondere Probleme macht bei Klassen mit Namen "Match" (wegen den neuen match expressions).
Und es ist wohl noch in der Diskussion, wie damit weiter umgegangen wird im Parser.
(Habe dort aber noch nicht in die Issues geschaut, wie gesagt nur so überflogen eine Diskussion in Slack.)

Dependabot wird versuchen zu updaten, und wir müssen ihm dann sagen, dass er zumindest die 4.7 ignorieren soll.

Reply:
https://github.com/nikic/PHP-Parser/issues/690
(wobei ich mir nicht 100% sicher bin, ob das genau auch das Problem ist bei den Fehlern bei uns.)